bsconfig.json

```diff
@@ -2,19 +2,13 @@
   "name": "ac-tune-maker",
   "version": "0.1.0",
   "sources": {
-    "dir" : "src",
-    "subdirs" : true
+    "dir": "src",
+    "subdirs": true
   },
   "package-specs": {
     "module": "commonjs",
     "in-source": true
   },
   "suffix": ".bs.js",
-  "bs-dependencies": [
-    "bucklescript-tea"
-  ],
-  "warnings": {
-    "error": "+101+8"
```

Comment:
do you happen to know what bsb does by default if this is excluded? I had to add the `+8` because I wanted it to error on non-exhaustive matches, and would like to preserve that behavior.

Reply:
Oops, I revert this change.